README.md

```diff
@@ -140,13 +140,53 @@ dependencies {
 
 ### Development
 
-#### Constructing Signature BaseString 
+#### Preparing HTTP Signature Token 
+
+Append this signature token into the Authorization header of the HTTP request
+
+Params:
+* realm
+* authPrefix - Authorization Header scheme prefix , i.e 'Apex_l2_eg'
+* httpMethod
+* urlPath
+* appId - App ID created in Gateway
+* secret - set to null for REST L2 SHA256WITHRSA
+* formList - to support parameter for form data if any
+* password
+* alias
+* fileName
+* nonce - set to null for random generated number
+* timestamp - set to null for current timestamp
+
+
+```java
+String realm = "<<your_client_host_url>>"
+String authPrefix = "<<authPrefix>>
+String httpMethod = "get"
+String url = "https://<<Target_URL>>/api/v1/?param1=first&param2=123";
+String certFileName = "certificates/ssc.alpha.example.com.p12";
+String password = "<<passphrase>>";
+String alias = "alpha";
+String appId = "<<appId>>";
+String secret = null;
+ApiList formList = null;
```

Comment:
Need to indicate formList is optional and also describe it?

Reply:
updated Readme.md